build-test.cmd

```diff
@@ -360,6 +366,20 @@ if errorlevel 1 (
     exit /b 1
 )
 
+xcopy /s /y "%CORE_ROOT_STAGE%" "%CORE_ROOT%"
```

Comment:
> it might be easier as a stop-gap to have each Helix job attempt to crossgen Core_Root .dll's before executing tests

Doesn't it mean that each job will be using its own core_root as opposed to a single one that you upload?